lib/src/sql/statements/live.rs

```diff
@@ -96,11 +122,13 @@ pub fn live(i: &str) -> IResult<&str, LiveStatement> {
 	Ok((
 		i,
 		LiveStatement {
-			id: Uuid::new(),
+			id: Uuid::new_v4(),
+			node: uuid::Uuid::new_v4(),
```

Comment:
Not sure why `node` is a random UUID or why either UUID is part of the `LiveStatement`.

Reply:
Soo.. I want to address this once we have actually landed the feature. All these changes became super fragile becomes of how many PRs branched off them. The tldr is
- API response serialises them differently
- storekey serialises them differently
- Conceptually, we don't want to muddy them; sql::uuid should be for queries only. Unfortunately, it is the most convenient way to deal with responses

So I could address that now. But I would rather address that in a separate PR that is clearly written with that in mind. But pushing back on that opinion is also valid - we shouldn't be merging code that we know is flawed.

Reply:
I don't object to expediting implementation in this way given it doesn't cause harm and we can fix it later :ok_hand: